modules/core/src/scenegraph/gltf/gltf-environment.js

```diff
@@ -0,0 +1,97 @@
+import GL from '@luma.gl/constants';
+import {Texture2D, TextureCube} from '../../webgl';
+import {loadImage} from '../../core/load-file';
+
+export default class GLTFEnvironment {
```

Comment:
I am not quite clear on what this class represents. A number of cube maps that together represents an advanced environment mapping? Could you try to write a doc page for it as a step towards articulating this? 

Also, how do we get additional such maps to use? (Apart from the one example we got through glTF demo)
* Are there libraries of such environments available? 
* Are they part of glTF files?
* Can we generate them? The classic technique is to set up 6 cameras and generate such maps from our scenegraph by rendering in each direction? Then we can create perfect reflections for any object in the scene.

Reply:
Yes, programs like [this one](https://github.com/derkreature/IBLBaker)